.ci/packaging.groovy

```diff
@@ -247,15 +251,13 @@ pipeline {
             }
           }
         }
-        /*
```

Comment:
Let's enable the E2E :)

Reply:
How about having them disabled until we use the new name-tag strategy for multiplatform?

Reply:
Ups, I saw the comment for maintaining the default tag meanwhile. LGTM!